auth/src/controllers/user.ts

```diff
@@ -225,9 +225,28 @@ userController.get('/list', async (req, res) => {
   }
 
   try {
-    const users = await UsersUseCases.getUserList(user)
+    const page = req.query.page ? parseInt(req.query.page as string) : undefined
+    const limit = req.query.limit
+      ? parseInt(req.query.limit as string)
+      : undefined
 
-    res.json(users)
+    // Validate pagination parameters
+    if (page !== undefined && !isNaN(page)) {
+      res.status(400).json({ error: 'Invalid page parameter' })
+
+      return
+    }
+
+    if (limit !== undefined && !isNaN(limit)) {
+      res.status(400).json({
+        error: 'Invalid limit parameter (must be a number)',
+      })
+      return
+    }
```

Comment:
This logic seems inverted, I think it should be something like:
```suggestion
    if (page !== undefined && (isNaN(page) || page < 1)) {
      res.status(400).json({ error: "Invalid page parameter" });
      return;
    }
    
    if (limit !== undefined && (isNaN(limit) || limit < 1 || limit > 100)) {
      res.status(400).json({
        error: "Invalid limit parameter (must be between 1 and 100)",
      });
      return;
    }
```